subgraph.yaml

```diff
@@ -9,8 +9,10 @@ dataSources:
     network: mainnet
     source:
       address: '0x5C69bEe701ef814a2B6a3EDD4B1652CB9cc5aA6f'
+#      address: '0xc0aee478e3658e2610c5f7a4a2e1777ce9e4f2ac' # sushi
       abi: Factory
       startBlock: 10000834
+#      startBlock: 10794229 # sushi
```

Comment:
same as above

Reply:
I added them to the Snapshot entity as well because it allowed me to calculate the gas costs of user's positions using 1 request instead of 2 (I need them to compute precise returns). I could first fetch the snapshots and then filter for tx but that is inefficient. But now that I am thinking about it a better solution might be to reference the Transaction entity directly.